functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -139,7 +142,8 @@ function Install-DbaMaintenanceSolution {
         [switch]$InstallJobs,
         [string]$LocalFile,
         [switch]$Force,
-        [switch]$EnableException
+        [switch]$EnableException,
+        [switch]$InstallParallel
```

Comment:
keep EnableException as the last one, sorry, forgot to mention it in the chat. Everything else looks good.